numba/tests/test_parfors.py

```diff
@@ -1957,6 +1957,100 @@ def test_impl(x, sz):
             return x
         self.check(test_impl, np.zeros((10, 10)), 3)
 
+    def test_prange_unknown_call1(self):
+        @register_jitable
+        def issue7854_proc(u, i, even, size):
+            for j in range((even + i + 1) % 2 + 1, size - 1, 2):
+                u[i, j] = u[i + 1, j] + 1
+
+        # issue7854
+        # Forbid fusion in unanalyzable call inside prange.
+        def test_impl(u, size):
+            for i in numba.prange(1, size - 1):
+                issue7854_proc(u, i, 0, size)
+            for i in numba.prange(1, size - 1):
+                issue7854_proc(u, i, 1, size)
+            return u
+
+        size = 4
+        u = np.zeros((size, size))
+        cptypes = (numba.float64[:,:], types.int64)
```

Comment:
```suggestion
        cptypes = (numba.float64[:, ::1], types.int64)
```
`u` is C ordered input?